cmd/rokka/cli/sourceimages.go

```diff
@@ -291,6 +336,21 @@ func init() {
 	sourceImagesListCmd.Flags().StringVar(&sourceImagesListOptions.Created, "created", "", "Created")
 	sourceImagesListCmd.Flags().StringVar(&sourceImagesListOptions.Sort, "sort", "", "Sort")
 
+	sourceImagesCopyAllCmd.Flags().IntVarP(
+		&copyAllOptions.Concurrency,
+		"concurrency",
+		"",
+		1,
```

Comment:
we could also default to `runtime.NumCPU()` or similar?

Reply:
It's more dependent on the backend than the client,  but 2 should be fine, since we usually have 2 servers (with more CPUs, but looks like that approx. hits our usual dynamodb limit)